### ISSUE TITLE
posts/2012/java-desktop-links-of-the-week-november-26/index

### ISSUE THREAD

**Issue by JonathanGiles** [JonathanGiles/jonathangiles.net-comments#14]
# Java desktop links of the week, November 26

[https://jonathangiles.net/posts/2012/java-desktop-links-of-the-week-november-26/](https://jonathangiles.net/posts/2012/java-desktop-links-of-the-week-november-26/)

**Comment by JonathanGiles** [comment]
**Auto-imported comment**, original author: antonis
**Comment posted on:** November 26, 2012

So, is Swing officially dead yet or not ?
I keep coming to this site knowing that the millions of lines of Swing code out there are actually 'Java Desktop' applications deserving a place in a such named series and hoping to find links to the work of my fellow Swing-ers.  Yet, Monday after Monday, all I can find is the most recent post of certain JavaFX-centric blogs needlessly repeated here (they are so easy to find if you are really looking for them anyway).

So please, if Swing is not officially dead, could you try to honor the work of those who created it, used it and are still maintaining it an using it for what it is:  a Java desktop framework and a very good at that. 

Or drop the misleading title ...

**Comment by JonathanGiles** [comment]
**Auto-imported comment**, original author: [Jonathan](http://JonathanGiles.net)
**Comment posted on:** November 26, 2012

Antonis, 

I would happily include Swing links in the weekly roundup if I could find them. However, as far as I know, there does not appear to be any Swing-related posts being created anymore. If you have recommended links then please email them to me, like many members of the JavaFX community do. It's not even a debate that all the links I post are easily found - all I'm doing is aggregating them in one spot for others who don't have the time to search for the links every now and then. By the way, the same argument applies to Swing posts - anybody can go out and search for them.

I should also note I'm a little disappointed to be accused of being misleading - I've been a proponent of the entire Java desktop ecosystem for a very long time, and try to cover all aspects of the 'scene'. It is not my job to make the news (nor should I note is it my job to even link to it - I do that out of the kindness of my heart). Accusing me of being misleading does not motivate me - it just makes me want to stop doing the links posts entirely (as it takes precious hours each week to do). 

So, in short, if you know of something worth blogging about, email me the link, or even better start publishing worthy blog posts and I will include them. Complaining that I'm not honoring the community of Java desktop developers really is not the best way to do things.

-- Jonathan
